brave/brave5/src/main/java/com/linecorp/armeria/client/brave/package-info.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 LINE Corporation
```

Comment:
using 2024 for new file?

Reply:
I believe this wasn't a new file because we used to have it but we moved it to another module and back here again. 😆  So let me just keep the year.